frontend/README.md

```diff
@@ -1,5 +1,5 @@
-### Important
```

Comment:
We're going to use yarn for frontend and npm for the backend. Is it okay for you to delete the package-lock.json file on the frontend and perform a yarn install on the frontend to get the correct dependencies in the yarn.lock file.